README.md

```diff
@@ -19,99 +51,105 @@ The following matrix lists supported versions of Kong for every release of Kong
 
 ## Get started
 
-Please check the [deployment documentation][1]
-
-## How it works?
-
-Using a standard [Kubernetes deployment][10], the Ingress controller runs multiple containers in a single pod. This allows us to define in a unit containing an [initContainer][11] to run Kong migrations, one container for the Kong admin API in control-plane mode and one container for the ingress controller itself. With this approach we simplify the deploy of the required components without user intervention.
-Once the deployment passes the readiness and liveness probes it means the Kong migrations ran and ingress controller can communicate with the Kong admin API.
-
-In a different deployment we run Kong in data-plane mode. This means the Kong instances only expose the proxy ports.
-By using this approach we can deploy and scale the data-plane with the requirements of your applications, i.e. using a daemonset, a deployment with affinity rules, etc.
+You can deploy Kong Ingress Controller on any
+Kubernetes cluster which Service of `type: LoadBalancer`.
 
-The next diagram shows how the components interact:
+You can use
+[Minikube](https://kubernetes.io/docs/setup/minikube/)
+on your local machine or use
+a hosted k8s service like
+[GKE](https://cloud.google.com/kubernetes-engine/).
 
-![kong components](docs/images/deployment.png "Kong Components")
+To setup Kong Ingress Controller in your k8s cluster, simply execute:
 
-## Custom annotations
+```shell
+kubectl apply -f https://bit.ly/kong-ingress
+```
 
-Please check the [annotations][7] document.
+This sets up Kong as your Ingress controller and
+will satisfy all Ingress resources that
+are created in your Kubernetes Cluster.
 
-## Annotation ingress.class
+Please refer our [deployment documentation][deployment-doc]
+for a detailed  introduction to Kong Ingress Controller
+and Ingress spec.
 
-If you have multiple Ingress controllers in a single cluster, you can pick one by specifying the `ingress.class` annotation, eg creating an Ingress with an annotation like
+## Seeking help
 
-```yaml
-metadata:
-  name: foo
-  annotations:
-    kubernetes.io/ingress.class: "gce"
-```
+Please search through the posts on
+[Kong Nation](https://discuss.konghq.com/c/kubernetes) as it's
+likely that another user has run into the same problem.
+If you don't find an answer, please feel free to post a question.
+If you've a feature request, please post on in
```

Comment:
using "you've" in this context is a little unusual - I'd suggest the full "If you have a..."

Reply:
"...please post on in..." -> "...please post in..."